endpoints-support.md

```diff
@@ -155,6 +155,16 @@
 |<sub>/rest/hypervisor-managers/{id}</sub>                                                |GET       | :heavy_minus_sign:    | :heavy_minus_sign:   |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
 |<sub>/rest/hypervisor-managers/{id}</sub>                                                |PUT       | :heavy_minus_sign:    | :heavy_minus_sign:   |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
 |<sub>/rest/hypervisor-managers/{id}</sub>                                                |DELETE    | :heavy_minus_sign:    | :heavy_minus_sign:   |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|     **ID Pool IPV4 Range**
+|<sub>/rest/id-pools/ipv4/ranges</sub>                                                    |POST      |  :white_check_mark: |  :white_check_mark: |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |  :white_check_mark:   |  :white_check_mark:   | :white_check_mark:   | :white_check_mark:  |
```

Comment:
I hope you validated this resource on OV 6.0, so please add new column "v2600" and mark the results for that version as well to avoid duplication of work in further OV 6.0 validations.

Reply:
done